common/lib/xmodule/xmodule/js/spec/capa/display_spec.coffee

```diff
@@ -723,6 +769,48 @@ describe 'Problem', ->
       expect($.postWithPrefix).toHaveBeenCalledWith '/problem/Problem1/problem_save',
           'foo=1&bar=2', jasmine.any(Function)
 
+    it 'tests that save does not enable the submit button when submit is originally disabled', (done) ->
+      deferred = $.Deferred()
+      self = this
+      curr_html = @problem.el.html()
+      runs = ->
```

Comment:
There is so much boilerplate in these test points.... If they can't be reduced down to something simple, please at least declare a common "postWithPrefix" spy that can be reused between test points.